tools/org.eclipse.n4js.hlc.base/src/org/eclipse/n4js/hlc/base/N4jscBase.java

```diff
@@ -931,18 +910,6 @@ private void initInjection(Properties properties) {
 		new N4JSStandaloneSetup().register(injector);
 		injector.injectMembers(this);
 
-		// STEP 2: set up language N4JSX
-
-		// By default, the ISetup classes generated by Xtext trigger the setup of parent languages (for example,
-		// setup of N4JSX triggers setup of N4JS, i.e. method #createInjectorAndDoEMFRegistration() in the generated
-		// class N4JSXStandaloneSetupGenerated invokes #doSetup() of class N4JSStandaloneSetup). In the following line,
-		// we have to avoid this implicit setup of N4JS, because N4JS was already initialized (another initialization
-		// would create a second injector for the language N4JS, which might lead to follow-up issues, e.g. multiple
-		// instances of singletons per language).
-		Injector n4jsxInjector = N4JSXStandaloneSetup.doSetupWithoutParentLanguages();
-		this.n4jsxFileExtensionsRegistry = n4jsxInjector.getInstance(FileExtensionsRegistry.class);
-		this.n4jsxFileBasedWorkspace = n4jsxInjector.getInstance(FileBasedWorkspace.class);
-		headless.setInstancesFromN4JSXInjector(n4jsxFileBasedWorkspace);
```

Comment:
in the Javadoc of the method containing this code, there is a `TODO` that is now also obsolete. Please remove.

Reply:
removed